src/components/Header.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from "react";
+import { Transition } from "@headlessui/react";
+import Image from "next/image";
+
+export default function Header({
+  index,
+}: {
+  index: string;
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleDropDown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div>
+      <div className="bg-white p-5 flex relative gap-2 font-normal space-x-10 text-sm items-center h-28 z-10">
+        <div className="gap-2 font-normal space-x-14 items-center flex">
+          <Image
+            src="/logo.png"
+            width={120}
+            height={120}
+            alt="test"
+            style={{ height: "100%", width: "auto" }}
+          />
+          <div className="dropdown relative w-max">
+            {headerSubSections.map((section, i) => (
+              <button
+                className="text-sm px-4 py-2.5 text-center inline-flex items-center hover:text-orange-600"
+                onClick={handleDropDown}
+                key={i}
+              >
+                {section.text}
+              </button>
+            ))}
+          </div>
+        </div>
+
+        <div className="gap-2 font-normal space-x-4 text-xs flex justify-end text-gray-500 p-2">
+          <a href="/#">Help</a>
+          <a href="/#">Find In-Store</a>
+          {headerIcons.map((images, index) => (
```

Comment:
images is plural, what it's representing is 1 image right?

Reply:
changed to image